packages/ui/src/components/va-slider/VaSlider.vue

```diff
@@ -344,6 +344,8 @@ export default defineComponent({
     }
 
     const moveStart = (e: MouseEvent | TouchEvent, index = currentSliderDotIndex.value) => {
+      e.preventDefault() // prevent page scrolling
```

Comment:
I'd use event modifier in template instead.

Reply:
Then you will need to add `.prevent` everywhere, where moveStart is used.